misc/theme-cicada/Makefile

```diff
@@ -1,5 +1,6 @@
 PLUGIN_NAME=		theme-cicada
-PLUGIN_VERSION=		0.1
+PLUGIN_VERSION=		0.3
+PLUGIN_REVISION=	1
```

Comment:
why not 0.2 ? and we don't need the revision bump :)

Reply:
fine with 0.3 but we don't need to bump revision, it's for when version does not change...